packages/itwin/grouping-mapping-widget/src/widget/components/provider/ManualUIComponent.tsx

```diff
@@ -0,0 +1,53 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import React from "react";
+import { Button, LabeledTextarea, Text } from "@itwin/itwinui-react";
+import { LoadingSpinner } from "../utils";
+import "./ManualUIComponent.scss";
+import type { CustomUIComponentProps } from "./CustomUIProvider";
+
+const ManualUIComponent = ({
```

Comment:
better name?

Reply:
ManualUIComponent is for manual query group builder. It follows the same name pattern as other. Detail explanation is in readme